op-devstack/devtest/testing.go

```diff
@@ -292,19 +290,8 @@ func (t *testingT) _TestOnly() {
 
 var _ T = (*testingT)(nil)
 
-// DefaultTestLogLevel is set to the TEST_LOG_LEVEL env var value, and defaults to info-level if not set.
-var DefaultTestLogLevel = func() slog.Level {
-	logLevel := os.Getenv("TEST_LOG_LEVEL")
```

Comment:
I'm fine with removing this, but can you update the `README.md` to not reference this `TEST_LOG_LEVEL`  env var anymore?